create-publication-from-doi/src/test/java/no/unit/nva/doi/fetch/service/IdentityUpdaterTest.java

```diff
@@ -89,13 +94,27 @@ public void shouldNotUpdateIdentifierWhenIdentifierAlreadyExistsWhenEnrichingPub
 
     @Test
     public void shouldNotAlterPublicationWhenMissingIdentityCallingEnrichPublicationCreators() {
-        var publication = createPublicationWithIdentity(null);
+        var publication = createPublicationWithIdentites(List.of());
         var cristinProxyClient = mock(CristinProxyClient.class);
         var updatedPublication = IdentityUpdater.enrichPublicationCreators(cristinProxyClient, publication);
 
         assertThat(updatedPublication, equalTo(publication));
     }
 
+    @Test
+    public void shouldNotEnrichContributorsWhenMoreThan10AreMissingCristinIdentifier() {
```

Comment:
I doubt that Cristin identifier will ever be populated in any case, so I'd maybe change the name here to `shouldNotEnrichContributorsWhenMoreThanTenUnverifiedContributorsExist` 

"Unverified" is the term for those authors that lack a Cristin ID.